main_program.py

```diff
@@ -1,3 +1,5 @@
+from coord import Coord
```

Comment:
This class does not need to be imported, it is already in robotsim.py. It could be left as is or removed.